### PR TITLE
Towncrier config

### LINKED ISSUE #141: Add towncrier
Add [towncrier](https://github.com/twisted/towncrier), see the [Noggin](https://github.com/fedora-infra/noggin) setup for an example.

### PR DESCRIPTION
**Pull request by james02135**
Added towncrier config data to the pyproject.toml file and two files into the new "news" directory based on Noggin configuration.

### CODE REVIEW

**Comment by james02135** [comment]
WIP - addded towncrier config data to the pyproject.toml file and two files into the new "news" directory based on Noggin configuration.

**Review by abompard — CHANGES_REQUESTED** [review]
We should avoid having copies of things when we can, so let's share the `get-authors.py` script and the template. They could be in a subdirectory of `/tools` for example.

File: datanommer.commands/datanommer/commands/__init__.py [hunk 1]
```diff
@@ -331,3 +331,23 @@ def formatter(key, val):
         results.append(formatter(result.category, result))
 
     click.echo("[%s]" % ",".join(results))
+
+
+# Set the version
+try:
+    import importlib.metadata  # pragma: no cover
```
Comment (StephenCoady) [inline datanommer.commands/datanommer/commands/__init__.py]: You don't need to add `pragma: no cover` to every line, you can add it to branch points instead and then everything in that branch won't be counted.

File: datanommer.consumer/datanommer/consumer/__init__.py [hunk 2]
```diff
@@ -43,3 +43,23 @@ def __call__(self, message):
         except Exception:
             m.session.rollback()
             raise
+
+
+# Set the version
+try:
+    import importlib.metadata  # pragma: no cover
```
Comment (StephenCoady) [inline datanommer.consumer/datanommer/consumer/__init__.py]: Same for this file.

**Review by StephenCoady — CHANGES_REQUESTED** [review]

File: datanommer.consumer/datanommer/consumer/__init__.py (hunk 2 above)
Comment (james02135) [reply datanommer.consumer/datanommer/consumer/__init__.py]: Should be all set

**Review by StephenCoady — APPROVED** [review]

**Pull request merged**